app/models/company.rb

```diff
@@ -99,6 +103,9 @@ class Company < ApplicationRecord
   has_many :reviews
   has_many :customers, through: :reviews
   has_many :quotes
+  
+  has_attached_file :avatar, styles: { medium: "300x300>", thumb: "100x100>" }, default_url: "/images/missing.png"
```

Comment:
Sure! We can do that! I'll be sure to make the changes this evening and add the pull request to Slack when it's ready.